lib/rexml/parsers/baseparser.rb

```diff
@@ -60,8 +60,11 @@ class BaseParser
       XMLDECL_PATTERN = /<\?xml\s+(.*?)\?>/um
       INSTRUCTION_START = /\A<\?/u
       INSTRUCTION_PATTERN = /<\?#{NAME}(\s+.*?)?\?>/um
+      INSTRUCTION_END = /#{NAME}(\s+.*?)?\?>/um
```

Comment:
Could you separate existing constants and newly added constants for easy to distinguish? For example:

```ruby
class BaseParser
  LETTER = '...'
  ...
  module Private
    INSTRUCTION_END = /#{NAME}(\s+.*?)?\?>/um
    # ...
  end
  private_constant :Private
  include Private
```

Reply:
OK
I See.